be/src/storage/lake/starlet_group_assigner.cpp

```diff
@@ -10,9 +10,13 @@
 
 namespace starrocks::lake {
 
-static const char* const kStarletPrefix = "staros_";
+static const char* const kStarletPrefix = "staros_s3://";
```

Comment:
use staros:// what the real filesystem is will be determined by shard_info.storage_path, it could be file:// or s3:// or cachefs:// or hdfs://